tests/recommend.yaml

```diff
@@ -279,13 +277,24 @@ mutualize:
     test-tilecloudchain:
       hosts:
         - name: main
-          host: demo.camptocamp.com
+          host: tiles.demo.example.com
+          tls:
+            secretName: main-tiles-tls
         - name: test
-          host: demo-test.camptocamp.com
+          host: tiles.demo-test.example.com
+          tls:
+            secretName: test-tiles-tls
       dir: tilegeneration
       tilecloudchain: true
       env:
         TEST: test
+  tiles:
+    service:
+      name: mutualise-tiles
+      port: 8080
```

Comment:
I'm used to see services published on port 80 even if pod use port 8080.
Maybe I miss something.